webpack/containers/Application/overrides.scss

```diff
@@ -19,6 +19,14 @@ td, th {
   word-wrap: break-word;
 }
 
+// needed because we have overflow set to auto for all td's and it affected the pf4 table
+.katello-pf4-table {
+  td {
+    max-width: 200px;
+    overflow: unset;
+    word-wrap: break-word;
+  }
+}
```

Comment:
Is this the best way to handle this? Check out the `overflow: auto` rule above on the `td, th`, that was causing the pf4 table to look bad and have scrollbars in the cells and the action menu partially hidden, so I'm unsetting it with a new class. But maybe there is a better fix.

Reply:
@jeremylenz  Looks like there is some [context here](https://github.com/Katello/katello/pull/8214/files#r310135247), it was added because of https://projects.theforeman.org/issues/25622 

@sjha4 Do you know of any updates to this issue?

Perhaps we can make this rule more specific, I'm not sure exactly how to go about handling this in the PR as a refactor will likely affect other areas, but will think it over. Maybe we can update it separately.

Reply:
@johnpmitsch : We can probably close that issue out since the overflow:auto fixed the issue of overflowing data in fields.

Reply:
I updated https://projects.theforeman.org/issues/25622 to reflect the current needs. I think we should refactor this class in a follow up PR since it was pre-existing, I'll add the issue to the CV page tracker.

Reply:
@johnpmitsch You can also set `overflow-x` and `overflow-y` separately if that will help.  I noticed the screen shot in the redmine was horizontal overflow, but the scrolling issue is vertical.